src/components/modules/paste.ts

```diff
@@ -324,13 +335,31 @@ export default class Paste extends Module {
 
         return;
       }
-
-      this.toolsTags[tag.toUpperCase()] = {
-        tool,
-      };
+      /** Sanitization configuration is string */
+      if (_.isString(tag)) {
+        this.toolsTags[tag.toUpperCase()] = {
+          tool,
+        };
+      }
+      /** Sanitization configuration is object */
+      if (_.isObject(tag)) {
+        this.toolsTags[Object.keys(tag)[0].toUpperCase()] = {
```

Comment:
What if developer will specify several tags?

```js
tags: [
  'tbody',
  {
    table: {
      width: true,
      height: true,
    },
  },
  {
    td: {
      colspan: true,
      rowspan: true
    },
    tr: {  // <-- the second tag
      height: true
    }
  },
]
```

will we allow that? If not, we need to write about it somewhere.

Reply:
Good point btw, we can implement multiple tags in the future

Reply:
@gohabereg, @neSpecc  I can right now but I need one tag which I can use parent to point out corresponding `tool` in `this.toolsTags` (which is mapping of tool tag name to tools). I can consider the first element for the same. what are your suggestions?

Reply:
this pr also include this too